### DISCUSSION TITLE
LosslessCut stopped saving my Mp4 files

### DISCUSSION THREAD

**Discussion by sujfuy** [mifi/lossless-cut#2242] [Q&A]
Haven't made any changes and I use this every week and it's awesome.... except today when I go to export it says it's processing 3 unprocessable files and there are 4 files in my output folder but no mp4... what could have happened?

**Comment by mifi** [comment]
Hi, can you provide more info, like screenshots of the output files, or which error message youre dgetting?

**Reply by sujfuy** [reply]
Thanks, I figured out it was something to do with attempting to change the file name before exporting...  couldn't figure it out, so I found it easier just to rename it after exporting.  But set it back to default and everything is fine again.

**Reply by mifi** [reply]
You can customise the output file name by clicking on it in the export dialog
